src/commands/ext-install.ts

```diff
@@ -114,17 +116,58 @@ async function installExtension(options: InstallExtensionOptions): Promise<void>
 /**
  * Command for installing a extension
  */
-export default new Command("ext:install <extensionName>")
-  .description("install an extension, given <extensionName> or <extensionName@versionNumber>")
+export default new Command("ext:install [extensionName]")
+  .description(
+    "install an extension, provide [extensionName] or [extensionName@versionNumber]; or omit and run with `-i` to see all available extensions."
+  )
```

Comment:
The option for `-i, --interactive` should be added here.

Reply:
I went back to API council and they approved adding `-i` globally.